### ISSUE TITLE
Compiler error: ',' expected (got "my_arg")

### ISSUE THREAD

**Issue by impopular-guy** [vlang/v#14001]
**V doctor:** 
```
OS: windows, Microsoft Windows 10 Home Single Language v19044 64-bit                                 
Processor: 8 cpus, 64bit, little endian, AMD Ryzen 5 3550H with Radeon Vega Mobile Gfx               
CC version: Error: exec failed (CreateProcess) with code 2: The system cannot find the file specified. 
 cmd: cc --version                                                                                     
                                                                                                       
getwd: C:\Users\dhrit\Desktop\ToReead\temp\vtest                                                                         
vmodules: C:\Users\dhrit\.vmodules                                                                     
vroot: C:\BIN\v                                                                                        
vexe: C:\BIN\v\v.exe                                                                                   
vexe mtime: 2022-04-10 15:40:52                                                                        
is vroot writable: true                                                                                
is vmodules writable: true                                                                             
V full version: V 0.2.4 3f90809.a0e7a46                                                                
                                                                                                       
Git version: git version 2.35.1.windows.2                                                              
Git vroot status: weekly.2022.13-109-ga0e7a46b                                                         
.git/config present: true                                                                              
thirdparty/tcc status: thirdparty-windows-amd64 9de1a49e
```

**What did you do?**
` v -cg run .\code.v`

code.v:
```v
struct MyStruct {
    var int
}

fn my_func(my_arg any) string {
    return "Success"
}

fn main(){
    d := MyStruct{120}
    println(my_func(d))
}
```

**What did you expect to see?**
`Success`
 
**What did you see instead?**
```
PS C:\Users\dhrit\Desktop\ToReead\temp\vtest> v -cg run .\code.v
C:/Users/dhrit/AppData/Local/Temp/v_0/code.4252710082346563978.tmp.c:450: warning: WINVER redefined
C:/Users/dhrit/AppData/Local/Temp/v_0/code.4252710082346563978.tmp.c:1909: error: ',' expected (got "my_arg")
builder error:
==================
C error. This should never happen.

This is a compiler bug, please report it using `v bug file.v`.

https://github.com/vlang/v/issues/new/choose

You can also use #help on Discord: https://discord.gg/vlang

PS C:\Users\dhrit\Desktop\ToReead\temp\vtest>
```

**Extra Info**
However, changing definition of `my_func` to this runs successfully.
```v
fn my_func(my_arg voidptr) string {
    return "Success"
}
```

**Comment by JalonSolov** [comment]
`any` is... not good.  There has been talk more than once about removing it from the language.

**Comment by impopular-guy** [comment]
> `any` is... not good. There has been talk more than once about removing it from the language.

I see.. thanks.
Should I close the issue then? Because `any` still exists in the language so ideally there shouldn't be any error.

**Comment by yuyi98** [comment]
Fixed.
```v
struct MyStruct {
    var int
}

fn my_func(my_arg any) string {
    return "Success"
}

fn main(){
    d := MyStruct{120}
    println(my_func(d))
}

PS D:\Test\v\tt1> v run .
./tt1.v:5:19: error: cannot use `any` type here, `any` will be implemented in V 0.4
    3 | }
    4 |
    5 | fn my_func(my_arg any) string {
      |                   ~~~
    6 |     return "Success"
    7 | }
```
